charm_test.go

```diff
@@ -4,27 +4,33 @@
 package charm_test
 
 import (
+	"bufio"
 	"bytes"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	stdtesting "testing"
+	"strings"
 
+	"github.com/juju/testing"
 	jc "github.com/juju/testing/checkers"
 	"github.com/juju/utils/fs"
 	gc "gopkg.in/check.v1"
+	"gopkg.in/juju/charm.v6"
 	"gopkg.in/yaml.v2"
 
-	"gopkg.in/juju/charm.v6"
+	coretesting "github.com/juju/juju/testing"
 )
 
 func Test(t *stdtesting.T) {
 	gc.TestingT(t)
 }
 
-type CharmSuite struct{}
+type CharmSuite struct {
+	coretesting.BaseSuite
```

Comment:
remove this BaseSuite - we only use it in juju/juju

Reply:
Using IsolationSuite